src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -138,7 +138,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_RESOURCES: '2019-07-01',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2019-06-01',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
-        ResourceType.MGMT_KEYVAULT: '2018-02-14',
+        ResourceType.MGMT_KEYVAULT: '2019-09-01',
```

Comment:
Many of KV features are using `2018-02-14` folder, including newly added Private Link, would this change causes multi-api problems?

Reply:
The `latest` profile should always use the latest API. As long as the live tests pass, we can assume it won't cause problems for new features like private link.